Domain Connect Spec Draft.adoc

```diff
@@ -1525,6 +1584,12 @@ Note: There are some templates that utilize CNAME values containing user
 identification for validation of domain ownership. For practical purposes 
 these values do not conflict with other services or sub-domains being 
 configured and are the only exception to this rule.
+It is however advisable, that such templates still define a constant
+part of a variable host name (a prefix) which will serve from one sidea
+as a name space lowering the probability of a conflict with different vendor,
+from the other side preventing phishing attack vectors which would impersonate
+the provider in order to push different data into DNS (which could be used
+to complete other mean of DNS-based verification).

```

Comment:
I get the idea of a prefix to lower the probability of a conflict.  That being said, few services in the wild that do this for verification actually use a prefix.  Many just use a GUID.

I don't get the way this prevents a phising attack.

Reply:
I already saw a template proposal with variables both in host and pointTo/value. In that extent one of the two shall be at least prefixed to avoid misuse. I'll re-formulate it to be clear.